apps/wing-console/console/server/src/utils/test-runner/simulator-manager.ts

```diff
@@ -0,0 +1,85 @@
+import { mkdtempSync } from "node:fs";
+import { tmpdir } from "node:os";
+import { join } from "node:path";
+
+import uniqueid from "lodash.uniqueid";
+
+import type { Simulator } from "../../wingsdk.js";
+import { createCompiler, type Compiler } from "../compiler.js";
+import { createSimulator } from "../simulator.js";
+
+const getSimulatorInstance = async (simfile: string) => {
+  const stateDir = mkdtempSync(
+    join(tmpdir(), `pwing-console-test-${uniqueid()}`),
+  );
```

Comment:
Maybe we could use node packages only. See: https://nodejs.org/api/fs.html#fspromisesmkdtempprefix-options

```js
import { mkdtemp } from 'node:fs/promises';
import { join } from 'node:path';
import { tmpdir } from 'node:os';

try {
  await mkdtemp(join(tmpdir(), 'foo-'));
} catch (err) {
  console.error(err);
}
```